akka-actor/src/main/scala/akka/util/Helpers.scala

```diff
@@ -130,12 +130,20 @@ object Helpers {
    * INTERNAL API
    */
   private[akka] final implicit class ConfigOps(val config: Config) extends AnyVal {
```

Comment:
ok, this is still `private[akka]` so we must be careful to not break bin compat if used from Alpakka

Perhaps we should introduce an additional comment to "INTERNAL API" for such cases. `OptionVal` is another candidate.

Reply:
In the end of the day Alpakka should refrain from using `private[akka]` API, I guess. Just wanted to set a standard how to deal with this.